client/src/components/FilterTable/FilterTable.stories.js

```diff
@@ -0,0 +1,164 @@
+import _ from "lodash";
+import React from "react";
+
+import { create_full_results_hierarchy } from "src/panels/panel_declarations/results/result_drilldown/result_hierarchies";
+
+import { text_maker } from "src/panels/panel_declarations/results/result_text_provider";
+
+import {
+  status_key_to_glossary_key,
+  ordered_status_keys,
+  result_color_scale,
+} from "src/panels/panel_declarations/results/results_common";
+
+import { toggle_list } from "src/general_utils";
+import {
+  IconCheck,
+  IconAttention,
+  IconNotApplicable,
+  IconClock,
+} from "src/icons/icons";
+
+import { FilterTable } from "./FilterTable";
+
+export default {
+  title: "Tables/FilterTable",
```

Comment:
This components not named very well, it's not a "Table". It's like a fancy checkbox control, so I'd put it in "Input/FilterTable".

Reply:
Done!